backends-velox/src/main/scala/org/apache/gluten/metrics/MetricsUtil.scala

```diff
@@ -219,6 +216,7 @@ object MetricsUtil extends Logging {
         })
 
     mutNode.updater match {
+      case MetricsUpdater.None =>
```

Comment:
Are we sure about this?

The code is likely something done by `MetricsUpdater.Todo`. I remember `MetricsUpdater.Todo` and `MetricsUpdater.None` don't share the same semantic. 

Did you check the UI? Are all the metrics still normal with this change?

Reply:
It doesn't seem to make any difference whether to add it or not.
Theoretically, the size of None's operatorMetrics is 0, and a judgment safety point is added.
Also, I didn't find any UI anomalies.

Reply:
I thought about it carefully later, and I should remove it, which can help us distinguish whether None is set incorrectly. If an UnsupportedOperationException is thrown later, it means that it should not be None